lxd/db/certificates.go

```diff
@@ -27,23 +27,28 @@ import (
 //go:generate mapper stmt -p db -e certificate update struct=Certificate
 //
 //go:generate mapper method -p db -e certificate List
-//go:generate mapper method -p db -e certificate Get
+//go:generate mapper method -p db -e certificate List-by-Fingerprint
+//go:generate mapper method -p db -e certificate Get-by-Fingerprint
```

Comment:
Will this just be a wrapped around `List` with a filter for the specified fingerprint, and perhaps some duplicate row checks (although technically not needed due to the unique index, although happy to have it for belt-and-braces)?

Reply:
All the `Get` methods just call their respective `List` method and check that the return size is 1. So `Get-by-Fingerprint`'s method would call `List-by-Fingerprint`'s method.